File: Backend/VehicleLoan/src/main/java/com/lti/vehicleloan/layer2/ApplicationFormDto.java
package com.lti.vehicleloan.layer2;

public class ApplicationFormDto {
	
	UserDetail userDetail;
	CarDetail carDetail;
	EmploymentDetail employmentDetail;
	LoanDetail loanDetail;
	AdvancedUserDetail advancedUserDetail;
	
	public UserDetail getUserDetail() {
		return userDetail;
	}
	public void setUserDetail(UserDetail userDetail) {
		this.userDetail = userDetail;
	}
	public CarDetail getCarDetail() {
		return carDetail;
	}
	public void setCarDetail(CarDetail carDetail) {
		this.carDetail = carDetail;
	}
	public EmploymentDetail getEmploymentDetail() {
		return employmentDetail;
	}
	public void setEmploymentDetail(EmploymentDetail employmentDetail) {
		this.employmentDetail = employmentDetail;
	}
	public LoanDetail getLoanDetail() {
		return loanDetail;
	}
	public void setLoanDetail(LoanDetail loanDetail) {
		this.loanDetail = loanDetail;
	}
	public AdvancedUserDetail getAdvancedUserDetail() {
		return advancedUserDetail;
	}
	public void setAdvancedUserDetail(AdvancedUserDetail advancedUserDetail) {
		this.advancedUserDetail = advancedUserDetail;
	}
}

File: Frontend/vehicleLoan/src/app/pojos/ApplicationFormDTO.ts
import { AdvancedUserDetail } from "./AdvancedUserDetail";
import { CarDetail } from "./CarDetail";
import { EmploymentDetail } from "./EmploymentDetail";
import { LoanDetail } from "./LoanDetail";
import { UserDetail } from "./UserDetail";

export class ApplicationFormDTO{
    userDetail: UserDetail;
    carDetail: CarDetail;
	employmentDetail: EmploymentDetail;
	loanDetail: LoanDetail;
	advancedUserDetail: AdvancedUserDetail;

}